atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManagerTest.java

```diff
@@ -74,7 +74,7 @@ public class SnapshotTransactionManagerTest {
     private static final String SETUP_TASK_METRIC_NAME =
             SnapshotTransactionManager.class.getCanonicalName() + ".setupTask";
     private static final String FINISH_TASK_METRIC_NAME =
-            SnapshotTransactionManager.class.getCanonicalName() + ".finishTask";
+            SnapshotTransactionManager.class.getCanonicalName() + ".runTaskThrowOnConflict";
```

Comment:
Note the metric name changed - from `finishTask` to `runTaskthrowOnConflict`. Curious if peeps think we should preserve the old name.

Reply:
I think this is fine, though thanks for flagging!